Readme.md

```diff
@@ -166,6 +166,13 @@ way to get them is via pip:
 
     pip install -e ".[dev]"
 
+For local development, you can use `uv` to create a virtual environment and
+install dependencies:
+
+    uv venv
+    source .venv/bin/activate
+    uv pip install -e ".[dev]"
+
```

Comment:
where does `uv` come from?

Reply:
this is unnecessary here. Everyone can use whatever wants to setup the env. I think a reason to add it, it could be if we had adapted it in our workflow and wanted to ensure that noone works with different packages. The instruction here, it has no really value.  It just another way but not related to the project. Can we remove it please?